src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultConsentViewModel.swift

```diff
@@ -75,7 +75,8 @@ class ExposureSubmissionTestResultConsentViewModel {
 									)
 								}
 							},
-						.space(height: 20)
+						.space(height: 20,
+							   color: .clear)
```

Comment:
Can't we set this to `.clear` by default?

Reply:
Thanks Nick. Yes we can & fixed it.